src/components/TodoContainer.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import TodosList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
+
+
+class TodoContainer extends React.Component {
+
+  state = {
+    todos: [
+      {
+        id: 1,
```

Comment:
- [ ] - [OPTIONAL ] Please replace the hardcoded `id` with `uuidv4()` to ensure id remains unique always